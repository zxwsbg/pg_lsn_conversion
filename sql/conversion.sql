CREATE EXTENSION pg_lsn_conversion;

SELECT convert_to_dec('3D7/40ABA220');

SELECT convert_to_lsn(4223037841952);