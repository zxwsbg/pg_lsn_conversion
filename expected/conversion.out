CREATE EXTENSION pg_lsn_conversion;
SELECT convert_to_dec('3D7/40ABA220');
 convert_to_dec 
----------------
  4223037841952
(1 row)

SELECT convert_to_lsn(4223037841952);
 convert_to_lsn 
----------------
 3D7/40ABA220
(1 row)

